public/theme/skeleton/search.twig

```diff
@@ -18,7 +18,7 @@
         {% setcontent block = "block/search-teaser" %}
 
         {# check if we have 'content'. If so, we know we have have a teaser to display. #}
-        {% if block is defined and block.hasField('content') %}
+        {% if block and block.content %}
```

Comment:
rename content to something more meaningful

Reply:
The "Blocks" ContentType has a `content` field. It's generic on purpose :-)